i18n/de/middleware/README.md

```diff
@@ -997,28 +997,48 @@ Folgende Systemvariablen stehen Ihnen zur Verwendung im Template zur Verfügung:
-| `reference_scu_swissbit_counter_{0-n}_url`       | wird verwendet um die Url einer bereits vorhandenen Swissbit usb SCU zu erhalten                                                  |
-| `reference_scu_cryptovision_counter_{0-n}_url`   | wird verwendet um die Url einer bereits vorhandenen cryptovision SCU zu erhalten                                                  |
-| `reference_scu_dieboldnixdorf_counter_{0-n}_url` | wird verwendet um die Url einer bereits vorhandenen diebold SCU zu erhalten                                                       |
-| `reference_scu_epson_counter_{0-n}_url`          | wird verwendet um die Url einer bereits vorhandenen epson SCU zu erhalten                                                         |
-| `reference_scu_atrust_counter_{0-n}_url`         | wird verwendet um die Url einer bereits vorhandenen atrust SCU zu erhalten                                                        |
+| `cashbox_id`                                     | Zufällige GUID                                                                                                                    |
+| `scu{0-9}_id`                                    | Zufällige GUID                                                                                                                    |
+| `helper{0-9}_id`                                 | Zufällige GUID                                                                                                                    |
+| `queue{0-9}_id`                                  | Zufällige GUID                                                                                                                    |
+| `queue{0-9}_id_base64withoutspecialchars`        | `{queue_id}`, konvertiert zu Base64 ohne Sonderzeichen. Wird oft als Kassenseriennummer verwendet (CashBoxIdentification)         |
+| `reference_scu_{type}_{key}_{value}_{attr}`      | Referenziert auf das angegebene Attribut einer existierenden SCU des gegebenen Typs und Schlüssel/Werte Paars                     |
 
 _Dynamische Werte werden in dieser Tabelle durch {} hervorgehoben._
 
+##### Typ
+
+`type` gibt den SCU Typ an, den Sie referenzieren möchten. Mögliche Werte sind
+
+* **atrust**
+* **cryptovision**
+* **dieboldnixdorf**
+* **epson**
+* **fiskaly**
+* **swissbit**
+* **swissbitcloud**
+
+
+
+##### Attribut
+
+`attr` gibt das SCU Attribut an, welches Sie referenzieren möchten. Mögliche Werte sind
+
+* **id** - referenziert auf die ID der jeweiligen SCU
+* **url** - referenziert auf alle konfigurierten URLs der jeweiligen SCU
+
+
+
+##### Schlüssel / Wert
+
+Für `key` und `value` kann jedes gültige Schlüssel/Wert Paar verwendet werden, das einer SCU des gewählten Typs **und**  in dem ausgewählten Standort zugewiesen wurde.
+
+
+
+> **Wichtig:** Standardmässig bekommen alle, über fiskaltrust.Shop erworbene, SCUs ein Feld names `counter` zugewiesen, welches einen für die jeweilige CashBox eindeutigen Integer Wert besitzt.
```

Comment:
Das `>` erzeugt in Markdown eigentlich einen Zitat-Bereich. Ich würde für solche Warnungen, Infos, etc. die Admonitions von Docusaurus verwenden:
https://docusaurus.io/docs/markdown-features/admonitions

Reply:
Quote ist grundsätzlich beabsichtigt gewesen, aber die erwähnten Blöcke sind ein guter Vorschlag.